include/saltatlas/common/data_reader.hpp

```diff
@@ -107,11 +108,18 @@ inline void read_points_helper(
     std::string line_buf;
     id_t        id = id_offsets[i];
     while (std::getline(ifs, line_buf)) {
-      point_t    point;
-      const auto ret = parser(line_buf, point);
-      if (!ret || point.empty()) {
```

Comment:
I also want to increase the value of `unreadable_lines` if `ret` is false or `point.empty()` is true.